### PR TITLE
Add a link in Host Details for more information on host + Set override policy for environment

### PR DESCRIPTION
**Pull request by sonia-y**

### CODE REVIEW

**Comment by sbaogang** [comment]
Look good to me, thanks! r+

**Comment by sbaogang** [comment]
@jinruh it would be great if we can also render the content natively in Teletraan, along with the link ( so people could follow the link to CMDB)

**Comment by sbaogang** [comment]
if this is not trivial, forget about it, this is good enough! Thanks!

**Comment by jinruh** [comment]
Thanks baogang, I would think to check in this first, and we can add more stuff later. 
r+

File: deploy-board/deploy_board/webapp/host_views.py [hunk 1]
```diff
@@ -88,6 +88,7 @@ def get(self, request, groupname, hostname):
             'asg_group': asg,
             'is_unreachable': is_unreachable,
             'pinterest': IS_PINTEREST,
+            'host_information_url': TELETRAN_HOST_INFORMATION_URL,
```
Comment (yujunglo) [inline deploy-board/deploy_board/webapp/host_views.py]: missing TELETRAN_HOST_INFORMATION_URL import from deploy_board.settings

**Pull request closed without merging**